### DISCUSSION TITLE
attaching additional custom metadata to pages

### DISCUSSION THREAD

**Discussion by benjamin-bergia** [requarks/wiki#6700] [Help / Questions] [answered]
Hello,

I have enjoyed working with wikijs quite a lot so far and I was wondering if there is any way to attach additional metadata to the pages that would be displayed in the UI? I have some simple scripts updating pages from other sources and I would really like to be able to store some metadata for this scripts to fetch. In my SQLite database I have noticed an `extra` JSON column. Would it be safe to hijack it and add some fields to the JSON it contains?

**Comment by NGPixel** [comment, marked as answer]
You can but there's no guarantee it won't be overwritten.

Page metadata is coming in v3.
